src/ws/hooks/chat.ts

```diff
@@ -42,36 +42,51 @@ export const configureWsChatHooks = (websocketClient: WebsocketClient) => ({
       const handler = (event: ChatEvent) => {
         if (event.kind === KINDS.ITEM) {
           const chatKey = buildItemChatKey(chatId);
-          const current = queryClient.getQueryData<ItemChatRecord>(chatKey);
+          const current = queryClient.getQueryData<ItemChat>(chatKey);
```

Comment:
I think this might be an error. Are we changing back to { id, messages: [] } for the chat response ? I thaight we returned only the messages ? there might have been an error in the types. It should be `ChatMessage[]` instead of `ItemChat`.